examples/tenant.yaml

```diff
@@ -148,6 +148,18 @@ spec:
   ## for details.
   podManagementPolicy: Parallel
 
+  ## serviceMetadata allows passing additional labels and annotations to MinIO and Console specific 
+  ## services created by the operator.
+  serviceMetadata:
+    minioServiceLabels:
+      label: minio-svc
+    minioServiceAnnotations:
+      v1.min.io: minio-svc
+    consoleServiceLabels:
+      label: console-svc
+    consoleServiceAnnotations:
+      v1.min.io: console-svc
+
   ## Add environment variables to be set in MinIO container (https://github.com/minio/minio/tree/master/docs/config)
```

Comment:
Is this still v1.min.io? @nitisht

Reply:
This was just and example to show how to add annotations, nevertheless updated now